uni/pubspec.yaml

```diff
@@ -72,13 +72,16 @@ dependencies:
   shimmer: ^3.0.0
   material_design_icons_flutter: ^7.0.7296
   flutter_dotenv: ^5.0.2
+  json_annotation: ^4.8.1
```

Comment:
Does this need to be a production dependency?

Reply:
It does, otherwise i could not use the "@JsonSerializable()" annotation on the classes